### DISCUSSION TITLE
Coronavirus acknowledgement

### DISCUSSION THREAD

**Discussion by BangDroid** [AnySoftKeyboard/AnySoftKeyboard#2747] [Q&A]
Is there a way to _acknowledge_ the Coronavirus icon that appears periodically, in such a way that it prevents future events?  
I'm totally on board with strategies to tackle the pandemic, however, the icon thing overlays word suggestions and it makes it difficult to use and it feels like it pops up every week.
I have enough reminders in my life (including my device) I don't really need my keyboard nagging me.
Cheers!

**Comment by menny** [comment]
Fair point. There isn't a way at the moment. I will work on something

**Reply by gkubed** [reply]
I am on the most recent release (v1.10.1794) and I still have the obnoxious reminder. I can't express how let down I am that this was even implemented in the first place.

**Comment by zaari** [comment]
I agree. Please, add an option to disable coronavirus reminders.

**Reply by szarkamax** [reply]
I like the keyboard and I thank you for it, but that COVID red virus icon is extremely annoying in the suggestion bar.

**Reply by geoGenius** [reply]
The Corona Virus hot key extremely annoying, especially when it distracts me when working. Please remove it! (Save the World some other way.)

**Comment by menny** [comment]
next release will no longer have this message

**Reply by tjfredsmith** [reply]
Why has it reappeared this week, again with no way to turn it off?!

**Reply by francois072** [reply]
I have been using anysoftkeyboard for a while. The virus reminders were there then later they dissapeared. Now when they came back I just switched to florisboard.

**Reply by ecster** [reply]
I love this keyboard and am so grateful for your work, I'm sure you added this because you care about the state of the world, but this issue was raised in March. I used a previous version on my old phone and installing it on my new phone gave me a bit of a shock, we are a year and a half into this pandemic... Please remove it.

**Reply by gkubed** [reply]
1.10.1794 and it's still there.

**Comment by menny** [comment]
Current ALPHA  does not have the COVID notice. Beta will come shortly (sometimes next week).

Stable will be release in a couple of weeks. I hope.

**Reply by athighnoon** [reply]
yes, many thx for trying to revert this crazy quackery. though such a harassment should not be brought upon users in first place. makes mixed feelings about this app and involved ppl

**Comment by acowegg** [comment]
Glad a fix is coming. Thank you for responding to this issue.

**Comment by acowegg** [comment]
Glad a fix is coming. Thank you for responding to this issue.
